extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,963 @@
+    const mortonIndex = morton(x, y, z);
+    if (level === 1) {
+        return 1 + mortonIndex;
+    }
+    // Level 1 Search
+    const levelOneCellMorton = mortonIndex >> 2;
+    if (sBuffer[1 + levelOneCellMorton] === 1) {
+        // Level 2... Search
+        return traverse(level, mortonIndex, 2, 5);
+    }
+    return -1;
+}
+
+function traverse(targetLevel, morton, currentLevel, levelOffset) {
+    const parentTileMorton = morton >> (2 ^ (targetLevel - currentLevel + 1));
+    const currentTileOffset = morton % 4;
+    const currentTileBufferOffset = levelOffset + jBuffer[currentLevel - 1].indexOf(parentTileMorton) + currentTileOffset;
+
+    if (targetLevel === currentLevel) {
+        return currentTileBufferOffset;
+    }
+
+    if (sBuffer[currentTileBufferOffset] === 1) {
+        return traverse(targetLevel, morton, currentLevel + 1, levelOffset + (jBuffer[currentLevel].length * tilingScheme));
+    } else {
+        return -1;
+    }
+}
+```
+
+### Samples
+
+#### Complete Octree
+
+##### Root Tileset - tileset.json
+
+```json
+{
+    "asset": {
+        "version": "2.0.0-alpha.0"
+    },
+    "geometricValue": 10000,
+    "extensions": {
+        "3DTILES_implicit_tiling": {
+            "boundingVolume": {
+                "box": [
+                    0, 0, 0,
+                    5, 0, 0,
+                    0, 5, 0,
+                    0, 0, 5
+                ]
+            },
+            "refine": "REPLACE",
+            "tilingScheme": "octree",
+            "contentExtension": "glb",
+            "tilesetExtension": "json",
+            "content": {
+                "levelOffset": 4,
+                "levelOffsetFill": 1
+            },
+            "subdivision": {
+                "completeLevels": 3
+            }
+        }
+    }
+}
+```
+
+##### Directory Structure
+
+```
+.
+└── tileset.json/
+    ├── L0/
+    │   └── 0/0/0/
+    │       └── 0/0/0.glb
+    ├── L1/
+    │   ├── 0/0/0/
+    │   │   └── 0/0/0.glb
+    │   ├── 0/1/0/
+    │   │   └── 0/1/0.glb
+    │   ├── 1/0/0/
+    │   │   └── 1/0/0.glb
+    │   ├── 1/1/0/
+    │   │   └── 1/1/0.glb
+    │   ├── 0/0/1/
+    │   │   └── 0/0/1.glb
+    │   ├── 0/1/1/
+    │   │   └── 0/1/1.glb
+    │   ├── 1/0/1/
+    │   │   └── 1/0/1.glb
+    │   └── 1/1/1/
+    │       └── 1/1/1.glb
+    ├── L2/
+    │   └── ...
+    └── ...    
+```
+
+#### Double Headed Quadtree (as 2 separate tilesets)
+
+##### Root 0 Tileset - tileset.json
+
+```json
+{
+    "asset": {
+        "version": "2.0.0-alpha.0"
+    },
+    "geometricError": 500,
+    "extensions": {
+        "3DTILES_implicit_tiling": {
+            "boundingVolume": {
+                "box": [
+                    0, 0, 0,
+                    5, 0, 0,
+                    0, 5, 0,
+                    0, 0, 5
+                ]
+            },
+            "tilingScheme": "quadtree",
+            "refine": "REPLACE",
+            "contentExtension": "glb",
+            "tilesetExtension": "json",
```

Comment:
Should this be `"glb"` or `".glb"`? I think the latter, but not totally sure.

Reply:
I think omitting the `.` is alright for now. Entwine, for examples, [does the same](https://entwine.io/entwine-point-tile.html#datatype).

Reply:
The difference is that their `dataType` is more like an enum and ours is the actual extension.

As a data point for the other side, [`path.extname`](https://nodejs.org/api/path.html#path_path_extname_path) includes the dot.

Reply:
Updated to include `.` in `contentExtension` and `tilesetExtension` properties.